app/views/meetings/_meeting_actions.html.haml

```diff
@@ -28,8 +28,8 @@
         = link_to "Can't make it anymore? Click here to cancel your spot.", meeting_cancel_path(meeting_id: @meeting.id, token: @invitation.token)
 
     - elsif @meeting.invitable && @meeting.not_full
-      = link_to meeting_invitation_path(@meeting), class: 'button round' do
-        = "Attend"
+      = link_to "Attend", meeting_invitation_path(@meeting, token: @invitation&.token),
+        class: "button round"
```

Comment:
nitpicking a bit, but can we please move this to the line above as this can earily be confused for nested content.

Reply:
Happy to, this is my Haml habit of breaking things up 😄

Reply:
OK, I just addressed this — sorry, I had forgotten about it 🙇